### DISCUSSION TITLE
Catch-up tv en Belgique

### DISCUSSION THREAD

**Discussion by Rxman06** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1154] [Q&A]
Bonjour j utilisé Catch-up tv chez moi sans problème j habite en France j ai voulu installer chez ma maman en Belgique (mis version bêta) mais pas d accès chaînes françaises tf1 France 2... Geo bloqué elle a juste m6 y a t il possibilité d avoir toutes les chaînes sans vpn payant merci d avance pour les réponses

**Comment by darodi** [comment]
@Rxman06 
Je confirme CUTVM se base sur les accès publiques des chaînes, si c'est geobloqué sur le site/stream officiel, c'est géobloqué dans CUTVM.

>  y a t il possibilité d avoir toutes les chaînes sans vpn payant ?

Rapidement, je vois 2 solutions
- Déménager en France
- Trouver un VPN non payant  

 :stuck_out_tongue_winking_eye:

En sachant que "si c'est gratuit, c'est vous le produit", je n'accorderais pas ma confiance à un VPN gratuit.
Tout le traffic passe par le VPN: collecte de données, man in the middle, ...

De Belgique, pour les chaînes fr

- Groupe M6 (W9, ...) : live non geo bloqué
- Groupe TF1 (TFX, ...): replay non geo bloqué
- Groupe France TV: tout géo bloqué.
- Groupe Canal : C8, ...  : cassé pour le moment, cf. https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/issues/1133
- RMC: bloqué
- Arte live bloqué, replay OK
- TV5: live bloqué, replay OK (TV5 monde plus, ...)

**Reply by Rxman06** [reply]
Merci pour ta réponse

**Reply by darodi** [reply]
Maintenant, si tu es en France, tu peux peut-être installer openvpn chez toi et faire le fournisseur VPN toi même pour ta maman?
